src/diffusers/models/unet_2d_condition_flax.py

```diff
@@ -214,6 +214,12 @@ def __call__(
             When returning a tuple, the first element is the sample tensor.
         """
         # 1. time
+        if not isinstance(timesteps, jnp.ndarray):
+            timesteps = jnp.array([timesteps], dtype=jnp.int32)
+        elif isinstance(timesteps, jnp.ndarray) and len(timesteps.shape) == 0:
+            timesteps = timesteps.astype(dtype=jnp.float32)
+            timesteps = timesteps[None]
```

Comment:
maybe use `expand_dims` as it will be cleaner.

Reply:
Fixed in 1facd9fa4b9fb164be779c9cb5feeb60c23c4895 !